common/src/main/java/com/revenuecat/purchases/common/FileHelper.kt

```diff
@@ -0,0 +1,86 @@
+package com.revenuecat.purchases.common
+
+import android.content.Context
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.InputStreamReader
+
+class FileHelper(
+    private val applicationContext: Context,
+) {
+    fun appendToFile(filePath: String, contentToAppend: String) {
+        val file = getFileInFilesDir(filePath)
+        file.parentFile?.mkdirs()
+        val outputStream = FileOutputStream(file, true)
```

Comment:
The way to do this in Java is to use "try with resource", since this is `Closable`, that way you avoid forgetting to close, or not closing it if an exception is thrown.

Looks like the `Kotlin` way of doing it is something like
```kotlin
val output = FileOutputStream(file, true)
output.use {
    outputStream.write(contentToAppend.toByteArray())
}
```

Reply:
You're completely right! Will change that, thanks!